.circleci/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+jobs:
+    build:
+        branches:  
+            ignore:   
+                - master
+        machine: true
```

Comment:
Is this needed !

Reply:
Yes @ksatchit I removed each instruction and checked, every time it threw an error. I believe the current version is the minimal config that we can go for.